nextflow_schema.json

```diff
@@ -66,10 +66,123 @@
+                    "description": "Specify whether to save the temporary files.",
+                    "fa_icon": "fas fa-save"
+                },
+                "taxa_classification_mmseqs_taxonomy_searchtype": {
+                    "type": "integer",
+                    "default": 2,
+                    "help_text": "Specify the type of alignment to be carried out between the query database and the reference MMseqs2 database. This can be set to '0' for automatic detection, '1' for amino acid alignment, '2' for translating the inputs and running the alignment on the translated sequences, '3' nucleotide based alignment and '4' for the translated nucleotide sequences alignment. More details can be found in the [documentation](https://mmseqs.com/latest/userguide.pdf).\n\n>  Modifies tool parameter(s):\n> - mmseqs taxonomy: `--search-type`",
+                    "description": "Specify the alignment type between database and query.",
+                    "fa_icon": "fas fa-align-center"
+                },
+                "taxa_classification_mmseqs_taxonomy_lcaranks": {
+                    "type": "string",
+                    "default": "kingdom,phylum,class,order,family,genus,species",
+                    "help_text": "Specify the taxonomic ranks to include in the taxonomic lineage column in the final `.tsv` file. For example, 'kingdom,phylum,class,order,family,genus,species'. More details can be found in the [documentation](https://mmseqs.com/latest/userguide.pdf).\n\n>  Modifies tool parameter(s):\n> - mmseqs taxonomy: `--lca-ranks`",
+                    "description": "Specify the taxonomic levels to display in the result table.",
+                    "fa_icon": "fas fa-stream"
+                },
+                "taxa_classification_mmseqs_taxonomy_taxlineage": {
+                    "type": "integer",
+                    "default": 1,
+                    "help_text": "This flag specifies whether the taxonomic lineage should be included in the output `.tsv` file. The taxonomic lineage is obtained from the internal module of `mmseqs/taxonomy` that infers the last common ancestor to classify the taxonomy. A value of '0' writes no taxonomic lineage, a value of '1' adds a column with the full lineage names prefixed with abbreviation of the lineage level, e.g. `k_Prokaryotes;p_Bacteroidetes;c_....;o_....;f_....;g_....;s_....,` while a value of '2' adds a column with the full NCBI taxids lineage,e.g. `1324;2345;4546;5345`. More details can be found in the [documentation](https://mmseqs.com/latest/userguide.pdf).\n\n>  Modifies tool parameter(s):\n> - mmseqs taxonomy: `--tax-lineage`",
+                    "description": "Specify whether to include or remove the taxonomic lineage.",
+                    "fa_icon": "fab fa-audible"
+                },
+                "taxa_classification_mmseqs_taxonomy_sensitivity": {
+                    "type": "string",
+                    "default": "5.0",
```

Comment:
why string and not number?

Reply:
@Darcy220606 Can we make this a number or does it have to be like this (also next parameter below)?